src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -5550,10 +5573,12 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						}
 					} else if (!strcasecmp(var, "rtcp-audio-interval-msec")) {
 						profile->rtcp_audio_interval_msec = switch_core_strdup(profile->pool, val);
+						sofia_set_pflag(profile, PFLAG_RTCP_AUDIO_INTERVAL_MSEC);
```

Comment:
We will need to have a separate configuration or flag to set rtcp audio interval for other channel

Reply:
Make sure to apply the same behavior for `export nolocal`. We don't want to set the interval msec applying for both channel.